docs/docs/migrating-from-v1-to-v2.md

```diff
@@ -175,22 +179,203 @@ export default props => (
 ```
 
 ## Rename `boundActionCreators` to `actions`
+
 `boundActionCreators` is deprecated in v2. You can continue using it, but it’s recommended that you rename it to `actions`.
 
+> TODO: document new actions - see [actions](/docs/actions)
+
 ## Rename `pathContext` to `pageContext`
+
 Similar to `boundActionCreators` above, `pathContext` is deprecated in favor of `pageContext`.
```

Comment:
Isn't this a breaking change? If so we should highlight that -- I don't believe boundActionCreators is.

Reply:
It's not — we pass both `pathContext` and `pageContext` as props